packages/babel-helper-create-class-features-plugin/src/index.ts

```diff
@@ -61,6 +64,9 @@ export function createClassFeaturePlugin({
     if (privateFieldsAsProperties !== undefined) {
       explicit.push(`"privateFieldsAsProperties"`);
     }
+    if (privateFieldsAsSymbols !== undefined) {
+      explicit.push(`"privateFieldsAsSymbols"`);
```

Comment:
Nit: we can export the `AssumptionName` type and annotate `explicit` as <code>&grave;"${AssumptionName}"&grave;[]</code>.

Reply:
Ah, so the (re-)export of `AssumptionName` was missing! And adding the quotes to the type is a nice trick! In my first attempt, I just added them afterwards (during output).